packages/fixie/src/main.ts

```diff
@@ -89,6 +89,39 @@ const currentPath = path.dirname(fileURLToPath(import.meta.url));
 const packageJsonPath = path.resolve(currentPath, path.join('..', 'package.json'));
 const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
 
+function errorHandler(error: any) {
+  if (error.response) {
+    // Error from a REST API call.
+    const url = error.response.config.url;
+    if (error.response.status == 401) {
+      term.red('❌ Could not authenticate to the Fixie API at ').green(`${url}.\n`);
+      if (process.env.FIXIE_API_URL) {
+        term('❌ FIXIE_API_URL is set to ').red(process.env.FIXIE_API_URL)('\n');
+        term('❌ Check to ensure that this is the correct API endpoint.\n');
+      }
+      if (process.env.FIXIE_API_KEY) {
+        term('❌ ').green('FIXIE_API_KEY')(' is set to ').red(process.env.FIXIE_API_KEY.slice(0, 12))('...\n');
+        term('❌ Check to ensure that this is the correct key.\n');
+      }
+    } else if (error.response.status == 400) {
+      term.red('❌ Client made bad request to ').green(`${url}.\n`);
```

Comment:
Maybe suggest that the user is on the latest version in this case?

Reply:
Good idea, done!